### PR TITLE
ccgx: Add some more plugin debugging

### PR DESCRIPTION
**Pull request by hughsie**
Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [ ] Code fix
- [ ] Feature
- [ ] Documentation

### CODE REVIEW

**Comment by hughsie** [comment]
@bleungatchromium this might be interesting to you too; I can share more details when Infineon have finished talking internally.

File: plugins/ccgx/ccgx.quirk [hunk 1]
```diff
@@ -141,3 +141,13 @@ Summary = Dock Management Controller Device
 CcgxDmcTriggerCode = 0x01
 CcgxImageKind = dmc-composite
 RemoveDelay = 732000
+
+# Caldigit ElementHub
```
Comment (superm1) [inline plugins/ccgx/ccgx.quirk:145]: This should be a separate commit

File: plugins/ccgx/fu-ccgx-dmc-devx-device.c [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+	/* add GUIDs */
+	fu_device_add_instance_strup(device,
+				     "TYPE",
+				     fu_ccgx_dmc_devx_device_type_to_string(status->device_type));
+	fu_device_add_instance_u8(device, "CID", status->component_id);
+	fu_device_add_instance_u16(device, "VID", fu_usb_device_get_vid(FU_USB_DEVICE(proxy)));
+	fu_device_add_instance_u16(device, "PID", fu_usb_device_get_pid(FU_USB_DEVICE(proxy)));
+	fu_device_build_instance_id(device, NULL, "USB", "VID", "PID", "CID", NULL);
+	fu_device_build_instance_id_quirk(device, NULL, "USB", "VID", "PID", "CID", "TYPE", NULL);
+	fu_device_build_instance_id_quirk(device, NULL, "USB", "VID", "PID", "CID", "VER", NULL);
```
Comment (superm1) [inline plugins/ccgx/fu-ccgx-dmc-devx-device.c:244]: This should only be if VER isn't null right?

Comment (hughsie) [reply plugins/ccgx/fu-ccgx-dmc-devx-device.c:244]: `fu_device_build_instance_id_quirk()` just skips the instance ID if any one of the keys is unset.

File: plugins/ccgx/ccgx.quirk (hunk 1 above)
Comment (hughsie) [reply plugins/ccgx/ccgx.quirk:145]: Agreed, done.

**Review by superm1 — APPROVED** [review]
This LGTM.  As part of reviewing it though, I noticed that the `RemoveDelay` key is the same for a lot of devices.  Should that maybe be the codified default too?

**Comment by hughsie** [comment]
> I noticed that the RemoveDelay key is the same for a lot of devices. 

I'll ask Infineon -- maybe we can set it automatically according to the CCG version or something.
